lib/CMakeLists.txt

```diff
@@ -84,7 +84,17 @@ set(mull_sources
   JunkDetection/CXX/Visitors/NegateConditionVisitor.cpp
   JunkDetection/CXX/ASTStorage.cpp
   JunkDetection/CXX/CompilationDatabase.cpp
-  Reporters/IDEReporter.cpp TestFrameworks/NativeTestRunner.cpp JunkDetection/CXX/Visitors/AndOrReplacementVisitor.cpp JunkDetection/CXX/Visitors/MathMulVisitor.cpp JunkDetection/CXX/Visitors/MathDivVisitor.cpp JunkDetection/CXX/Visitors/ReplaceCallVisitor.cpp)
+  JunkDetection/CXX/Visitors/AndOrReplacementVisitor.cpp
+  JunkDetection/CXX/Visitors/MathMulVisitor.cpp
+  JunkDetection/CXX/Visitors/MathDivVisitor.cpp
+  JunkDetection/CXX/Visitors/ReplaceCallVisitor.cpp
+
+  Reporters/IDEReporter.cpp
+  Reporters/MutationTestingElementsReporter.cpp
+  Reporters/SourceManager.cpp
+
+  TestFrameworks/NativeTestRunner.cpp
+)
```

Comment:
Nice one, I didn't pay attention to this file, seems like CLion could do a better job here.